.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default owner
+* @elastic/apm-agent-python
+
```

Comment:
```suggestion
```

Reply:
When PRs are not risky I tend to merge them but I'm not actively using opbeans-loadgen.